common/launch_test.go

```diff
@@ -3,46 +3,42 @@ package common
 import (
 	"testing"
 
-	"github.com/grafana/xk6-browser/k6ext/k6test"
-
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/grafana/xk6-browser/k6ext/k6test"
 )
 
 func TestLaunchOptionsParse(t *testing.T) {
 	t.Parallel()
 
-	testCases := []struct {
-		name   string
+	for name, tt := range map[string]struct {
 		opts   map[string]any
-		assert func(*testing.T, *LaunchOptions)
+		assert func(testing.TB, *LaunchOptions)
```

Comment:
Do we need to leave this as `TB`? What value do the benchmarks give us?

Reply:
It's because of the linter 🤦‍♂️